tools/export/GettingStarted.html

```diff
@@ -3,14 +3,14 @@
     <head>
         <meta charset="UTF-8">
         <meta http-equiv="refresh"
-              content="1;url="https://developer.mbed.org/handbook/Getting-Started-mbed-Exporters>
+              content="1;url="https://os.mbed.com/docs/v5.6/tools/exporting.html>
```

Comment:
no go on hardcoding this to a specific version...

Reply:
@sg- I'm going to update this to the "latest" link when we have that link.